src/styles.js

```diff
@@ -90,97 +94,115 @@ const loggedInStyles = css`
 `
 
 const accordionStyles = css`
-h2 {
-  font-weight: 700;
-  letter-spacing: 1px;
-  display: block;
-  background-color: white;
-  margin: 0;
-  cursor: pointer;
-  @extend .no-select;
-}
-
-p {
-  margin: 0;
-  font-size: 18px;
-}
-
-div[name='accordion'] {
-  position: relative;
-  overflow: hidden;
-  max-height: 800px;
-  opacity: 1;
-  transform: translate(0, 0);
-  margin-top: 14px;
-  z-index: 2;
-}
+  h2 {
+    font-weight: 700;
+    letter-spacing: 1px;
+    display: block;
+    background-color: white;
+    margin: 0;
+    cursor: pointer;
+    @extend .no-select;
+  }
 
-ul {
-  list-style: none;
-  perspective: 900;
-  padding: 0;
-  margin: 0;
-}
+  p {
+    margin: 0;
+    font-size: 18px;
+    font-style: italic;
+  }
 
-li {
-  position: relative;
-  padding: 0;
-  margin: 0;
-  padding-bottom: 4px;
-  padding-top: 18px;
-  border-top: 1px dotted grey;
-
-  i {
-    position: absolute;
-    transform: translate(-6px, 0);
-    margin-top: 16px;
-    right: 0;
-
-    &:before,
-    &:after {
-      content: '';
-      position: absolute;
-      background-color: black;
-      width: 3px;
-      height: 9px;
-    }
+  div[name='accordion'] {
+    position: relative;
+    overflow: hidden;
+    opacity: 1;
+    transform: translate(0, 0);
+    margin-top: 14px;
+    z-index: 2;
+  }
 
-    &:before {
-      transform: translate(-2px, 0) rotate(45deg);
+  #line367 dt.key,
```

Comment:
Isn't there another way to do this by targeting a parent element?

Reply:
I simplified it to just `#line367 dt` I figured this would probably end up becoming obsolete and replaced anyway, once the Accordion gets re-done.